packages/webapp/pages/posts/[id]/seo.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { ReactElement } from 'react';
```

Comment:
Is this file meant to be in `pages` since it will try to create `posts/:id/seo` page handler. Probably move it to utils?

Reply:
Maybe that is why the build is failing as well.